src/docfx.website.themes/default/partials/affix.tmpl.partial

```diff
@@ -1,7 +1,25 @@
 {{!Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.}}
 
 <div class="hidden-sm col-md-2" role="complementary">
-  <nav class="bs-docs-sidebar hidden-print hidden-xs hidden-sm affix" id="affix">
-  <!-- <p><a class="back-to-top" href="#top">Back to top</a><p> -->
-  </nav>
+  <div class="sideaffix">
+    {{^_disableContribution}}
```

Comment:
>_disableContribution [](start = 7, length = 20)

new pages are published to [https://hellosnow.github.io/docfx/](https://hellosnow.github.io/docfx/), please help to review the styles